scheduler/scripts/utils/get_job_overheads.py

```diff
@@ -0,0 +1,77 @@
+import argparse
+import datetime
+import os
+import re
+
+def get_job_overhead(timeline):
+    computation_time = 0.0
+    total_time = 0.0
+    current_dispatch_start_time = None
+    current_compute_start_time = None
```

Comment:
Add a comment stating the limitations (can't handle multi-job and multi-GPU traces because job dispatch can be overlapped with previous job execution, for example?)

Reply:
Added a longer description